changelog/fix-3845-update-breaking-e2e-tests-grouped-settings

```diff
@@ -0,0 +1,5 @@
+Significance: patch
```

Comment:
A change log entry may not be required for the change since it's related to devops. Also the change is based on the Dev Tools plugin which is not public as well

Reply:
Indeed, I removed it now. Thanks